src/plugins/redis.ts

```diff
@@ -45,7 +45,7 @@ export default fastifyPlugin(
 			},
 		);
 
-		redis.remember = async (key, ttl, callback) => {
+		redis.remember = async (key, ttl_seconds, callback) => {
```

Comment:
Isn't this milliseconds?

Reply:
https://redis.github.io/ioredis/classes/Redis.html#setex